pages/_document.tsx

```diff
@@ -4,7 +4,7 @@ export default function Document() {
   return (
     <Html lang="en">
       <Head>
-        <link href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800&display=swap" rel="stylesheet" />
+        <link href="https://fonts.googleapis.com/css?family=Inter&display=optional" rel="stylesheet" />
```

Comment:
Is this on purpose? Because I believe some font weights are not being imported unless specified